packages/browser-wallet/src/popup/pages/Account/i18n/en.ts

```diff
@@ -52,6 +52,13 @@ const t = {
         fee: 'Estimated transaction fee',
     },
     tokens: {
+        prompt: {
+            description: 'dApp would like to add tokens.',
+            descriptionAllExisting: 'dApp would like at add the following tokens. They are all already added though.',
```

Comment:
Do we want to use the word "dApp" here, or should we put in the app title/url instead? As a user, I guess you reveal that you've added the tokens to your wallet in the reponse, so it can be relevant to know who's issuing the request.

Reply:
It is supposed to be the url.